tools/c7n_mailer/tests/test_utils.py

```diff
@@ -26,6 +26,21 @@ def test_strip_prefix(self):
         self.assertEqual(utils.strip_prefix('', 'aws.'), '')
 
 
+def test_config_defaults():
+    config = {}
+    utils.setup_defaults(config)
+    for k, v in list(config.items()):
+        if v is None:
+            config.pop(k)
```

Comment:
Good call at popping off the Nones I was looking at a similar test last night and had to assert equal like 4 or 5 of them.

Reply:
yeah i was wondering about asserting the set of empty ones though, but this suffices for now